src/main/java/org/mockito/Mockito.java

```diff
@@ -95,15 +95,23 @@
  * <h3 id="0.1">0.1. <a class="meaningful_link" href="#mockito-android">Mockito Android support</a></h3>
  *
  * With Mockito version 2.6.1 we ship "native" Android support. To enable Android support, add the `mockito-android` library as dependency
- * to your project. This artifact is published to the same mockito organization:
+ * to your project. This artifact is published to the same Mockito organization and can be imported for Android as follows:
  *
  * <pre class="code"><code>
- * repositories { jcenter() }
- * dependencies { testCompile "org.mockito:mockito-android:2.+" }
+ * repositories {
+ *   jcenter()
+ * }
+ * dependencies {
+ *   androidTestCompile "org.mockito:mockito-core:+"
+ *   androidTestCompile "org.mockito:mockito-android:+"
```

Comment:
I like that the example uses dynamic versions. It's easier for us to maintain and also potentially gets us users that are always on the latest version.

Why does the user need to declare both dependencies? mockito-android depends on mockito-core and pulls it as transitive dependency. I would recommend to just depend on mockito-android.